contracts/mocks/InitializableMock.sol

```diff
@@ -105,7 +105,9 @@ contract ReinitializerMock is Initializable {
     }
 
     function doStuff() public onlyInitializing {
-        counter++;
+        unchecked {
+            counter++;
+        }
```

Comment:
we don't care about the gas efficiency of mocks. Lets leave them as they are for the sake of readability.